runelite-client/src/main/java/net/runelite/client/plugins/worldmap/TeleportLocationData.java

```diff
@@ -138,6 +138,7 @@
 	PHARAOHS_SCEPTRE_JALSAVRAH(TeleportType.OTHER, "Pharaoh's Sceptre", "Jalsavrah (Pyramid Plunder)", new WorldPoint(3288, 2795, 0), "pharaohs_sceptre_teleport_icon.png"),
 	PHARAOHS_SCEPTRE_JALEUSTROPHOS(TeleportType.OTHER, "Pharaoh's Sceptre", "Jaleustrophos (Agility Pyramid)", new WorldPoint(3341, 2827, 0), "pharaohs_sceptre_teleport_icon.png"),
 	PHARAOHS_SCEPTRE_JALDRAOCHT(TeleportType.OTHER, "Pharaoh's Sceptre", "Jaldraocht (Desert Treasure Pyramid)", new WorldPoint(3232, 2897, 0), "pharaohs_sceptre_teleport_icon.png"),
+	CAMULET_TEMPLE(TeleportType.OTHER, "Camulet", "Enhakra's Temple", new WorldPoint(3190, 2923, 0), "camulet_teleport_icon.png"),
```

Comment:
I believe this should be `Enakhra` instead of `Enhakra`.

Reply:
Good spot